database/entity/0055-add_communities_table/Community.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column } from 'typeorm'
+
+@Entity('communities')
+export class Community extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({ name: 'public_key', type: 'binary', length: 64, default: null, nullable: true })
+  publicKey: Buffer
+
+  @Column({ name: 'api_version', length: 10, nullable: false })
+  apiVersion: string
+
+  @Column({ name: 'endpoint', length: 255, nullable: false })
+  endPoint: string
```

Comment:
either:

```suggestion
  @Column({ name: 'end_point', length: 255, nullable: false })
  endPoint: string
```

or:

```suggestion
  @Column({ name: 'endpoint', length: 255, nullable: false })
  endpoint: string
```

Reply:
if i have the choice, then i prefere the first suggestion... ;-)